controllers/rcj_soccer_team_blue/rcj_soccer_robot.py

```diff
@@ -191,5 +204,14 @@ def get_sonar_values(self) -> dict:
             "back": self.sonar_back.getValue(),
         }
 
+
+    def kick_ball(self, state, velocity=20):
+        if state == 1:
+            self.kicker_motor.setVelocity(velocity)
+            self.kicker_motor.setPosition(0.042)
```

Comment:
Hello @mrshu,

Yes, the value of 0.042 here represents the furthest position that the kicker could move to without moving beyond the frame of the robot (the greater the value, the further the center of the kicker is away from the center of the robot). 

When we were making the kicker in the Webots world, we only made sure that the kicker position had to be between 0 and 0.042. While this means that the kicker could be moved to positions between 0 and 0.042, the kicker would not be kicking rthe ball as hard in those cases. 

If you would like, we could add a parameter to the `kick_ball` function that would allow the kicker to be moved to specific positions between 0 and 0.042. 

Thank you!

Team Reset